### PR TITLE
(fix) Langfuse v2 renamed a few things.

### LINKED ISSUE #2857: [Bug]: Broken times in Langfuse?
### What happened?

Not sure why, but it seems like timing in Langfuse (from LiteLLM) got borked after I updated to v1.34.25.

image

### Relevant log output

_No response_

### Twitter / LinkedIn details

https://www.linkedin.com/in/davidmanouchehri/

### PR DESCRIPTION
**Pull request by Manouchehri**
Fixes #2857.

### CODE REVIEW

**Comment by Manouchehri** [comment]
This is what Langfuses looks like after applying this patch:

image

**Comment by Manouchehri** [comment]
I've been running this for a few days with no problems. :)

File: litellm/integrations/langfuse.py [hunk 1]
```diff
@@ -324,13 +326,17 @@ def _log_langfuse_v2(
...
+            system_fingerprint = response_obj.get("system_fingerprint", None)
+            if system_fingerprint is not None:
+                optional_params["system_fingerprint"] = system_fingerprint
+
             generation_params = {
                 "name": generation_name,
                 "id": metadata.get("generation_id", generation_id),
-                "startTime": start_time,
-                "endTime": end_time,
+                "start_time": start_time,
```
Comment (ghost) [inline litellm/integrations/langfuse.py:336]: @Manouchehri is this dependent on a specific minor version of their sdk? or was this on their api  (i.e. version independent)

Comment (Manouchehri) [reply litellm/integrations/langfuse.py:336]: It's always been like this for Langfuse's v2 SDK. The old code here in LiteLLM was always wrong for v2 I think, not sure why it worked at all to begin with tbh. 😅 

https://langfuse.com/docs/sdk/python/low-level-sdk#generation

**Pull request merged**